doc/source/train/user-guides/checkpoints.rst

```diff
@@ -309,6 +316,77 @@ Lower-performing checkpoints are deleted to save storage space. By default, all
     please ensure that the metric is always reported together with the checkpoints.
 
 
+.. _train-validating-checkpoints:
+
+Validating checkpoints asynchronously
+-------------------------------------
+
+You can also asynchronously validate checkpoints that you :func:`~ray.train.report` as follows:
+
+* Define your own ``validate_fn`` that takes a :class:`~ray.train.Checkpoint` to validate
+  and an optional ``validate_config`` dictionary, which contain arguments needed for validation
+  such as the validation dataset, and returns a dictionary of metrics from the validation.
+  We recommend performing the validation with an eval-only :ref:`Trainer <train-overview-trainers>` or with
+  :func:`~ray.data.Dataset.map_batches` - see :ref:`train-validate-fn` for more details.
+* Call :func:`~ray.train.report` with your ``validate_fn`` and a ``validate_config`` dict.
+  Ray Train runs your ``validate_fn`` with the ``validation_config`` and ``checkpoint``
+  in a new Ray task. When that task completes, Ray Train associates the metrics returned by
+  ``validate_fn`` with that ``checkpoint``.
+
+The main benefits of validating with :func:`~ray.train.report` over performing the validation
+in the training loop include:
+
+* Running validation in parallel with the training loop
+* Running validation on different hardware than training
+* Leveraging :ref:`vms-autoscaling` to rent user-specified machines only for the duration of the validation
```

Comment:
nit: Rename this so it reads better.

```suggestion
* Leveraging :ref:`Autoscaling <vms-autoscaling>` to rent user-specified machines only for the duration of the validation
```

<img width="836" height="44" alt="Image" src="https://github.com/user-attachments/assets/8d33edfd-897c-4b6e-bdcd-9a7b45b29c4c" />